docs/pages/includes/database-access/tctl-auth-sign-3-files.mdx

```diff
@@ -0,0 +1,60 @@
+{{ format="db" }}
+Teleport uses mutual TLS authentication with self-hosted databases. These
+databases must be able to verify certificates presented by the Teleport Database
+Service. Self-hosted databases also need a certificate/key pair that Teleport
+can verify.
+
+By default, the Teleport Database Service trusts certificates issued by a
+certificate authority managed by the Teleport Auth Service. You can either:
+- Configure your self-hosted database to trust this CA, and instruct Teleport to
+  issue a certificate for the database to present to the Teleport Database
+  Service.
+- Configure the Database Service to trust a custom CA.
+
+To configure the database to trust the Teleport CA and issue a certificate for
+the database, follow these instructions on your workstation:
+
+1. To use `tctl` from your workstation, your Teleport user must be allowed to
+   impersonate the system role `Db` in order to be able to generate the database
+   certificate. Include the following `allow` rule in in your Teleport user's
+   role:
+
+   ```yaml
+   allow:
+     impersonate:
+       users: ["Db"]
+       roles: ["Db"]
+   ```
+
+1. Export Teleport's certificate authority and a generate certificate/key pair.
+   This example generates a certificate with a 1-year validity period.
+   `db.example.com` is the hostname where the Teleport Database Service can
+   reach the {{ dbname }} server.
+
+   ```code
+   $ tctl auth sign --format={{ format }} --host=db.example.com --out=server --ttl=2190h
+   ```
+
+   (!docs/pages/includes/database-access/ttl-note.mdx!)
+
+   The command creates 3 files: `server.cas`, `server.crt` and `server.key`.
+
+If the {{ dbname }} database already has a CA that it uses to sign certificates
+, you only need to export a Teleport CA certificate for the database to
+authenticate traffic from the Teleport Database Service. You do not need to
+enable `Db` impersonation privileges.
+
+1. Replace <Var name="example.teleport.sh:443" /> with the host and web port of
+   the Teleport Proxy Service in your cluster. Run the following command on your
+   workstation:
+
+   ```code
+   $ tctl auth export --type=db-client --auth-server=<Var name="example.teleport.sh:443" /> > server.cas
```

Comment:
this `server.cas` cannot be used by the db server directly. Instead, we should append our db client CA to their existing `server.cas`

Reply:
👍 I've updated the partial. I'd also be interested in knowing why this is!

Reply:
It just needs to have both Teleport's CA and their custom CA, otherwise their custom CA client certs will get rejected.

Whether that be 
"append your custom CA to server.cas"
or
"append Teleport's CA to your CA file"

Both will work.

Reply:
Your updated version works perfectly btw 👍